docs/source/Knowledge_Base/crashreport.rst

```diff
@@ -48,12 +48,12 @@ ATLauncher
 5. Click on the ``Create new paste`` button and copy the web link.
 6. Paste the link in the forum thread/discord/ticket.
 
-Twitch Launcher
+Curseforge Launcher
 ===============
```

Comment:
Please add the missing `=` here. Needs to be the same length as the text above

Reply:
Ah thanks for the info!

I think i got that added correctly in the new commit. Still getting used to working with Github.